### DISCUSSION TITLE
Reporting system

### DISCUSSION THREAD

**Discussion by amoush84** [osTicket/osTicket#5968] [General]
Hi 
I am looking for reporting system for osticket

**Comment by mlipok** [comment]
CNN? 
Be more specific.
Btw.
You should start by ask on forum.

**Comment by ntozier** [comment]
@amoush84 this does not appear to be a bug or issue report.

If you have a general use or non bug report issue you would want to ask on the [forums](https://forum.osticket.com/). 

That being said I maintain a list of [Resources for osTicket](https://forum.osticket.com/d/92286-resources-for-osticket) list on the forums and on my personal site.  You would likely find what you are looking for on that list.
